source/content/guides/multisite/06-search-replace.md

```diff
@@ -103,27 +106,29 @@ Environments that need to be replaced are defined and paired in the `sites.yml`
 
 1. Commit the  `sites.yml` file in the `private/sites.yml` in the site’s Git repository.
 
-### Subdomain to Subdirectory Conversion
+## Subdomain to Subdirectory Multisite Conversion
 
-Sites can also be configured to use subdomain Multisite in the Live environment, and subdirectory in all other instances.
+Sites can also be configured to use Subdomain Multisite in the Live environment, and Subdirectory in all other environments.
 
 To configure this:
 
-1. Make `SUBDOMAIN_INSTALL` conditional in the `wp-config.php` file:
+1. In your `pantheon.yml` file, set the `search_replace` value to `convert`.
+
+  ```yaml:title=pantheon.yml
+  search_replace: convert
+  ```
+
+1. Open your `wp-config.php` file and locate the `SUBDOMAIN_INSTALL` key.
+
+1. Set the `SUBDOMAIN_INSTALL` value to conditional:
 
     ```php:title=wp-config.php
     if ( ! empty( $_ENV['PANTHEON_ENVIRONMENT'] ) && $_ENV['PANTHEON_ENVIRONMENT'] ==  'live' ) {
       define( 'SUBDOMAIN_INSTALL', true );
     }
     ```
 
-1. Set `convert_to_subdirectory: true` in the `sites.yml` file.
```

Comment:
Do we want a note/alert in the docs to the effect of "this used to work and no longer does", or is that more confusing?

Reply:
Naw. I'll communicate to the EA folks.